server/controllers/languagesController.ts

```diff
@@ -4,13 +4,13 @@ import { AuditService, Page, PostAction } from '../services/auditService'
 import logger from '../../logger'
```

Comment:
I've moved the duplicate language logic out of the client side js into the controller, enabling us to reuse the common `autocomplete.js` code for autocomplete.